python/pyspark/sql/session.py

```diff
@@ -73,7 +73,7 @@ class SparkSession(SparkConversionMixin):
 
     A SparkSession can be used create :class:`DataFrame`, register :class:`DataFrame` as
     tables, execute SQL over tables, cache tables, and read parquet files.
-    To create a SparkSession, use the following builder pattern:
+    To create a :class:`SparkSession`, use the following builder pattern:
```

Comment:
This is fine, but do we do it consistently? if that would be a big change, I wonder how much it's worth it

Reply:
Yeah, I agree.

Reply:
It seems to be already done in most classes below sql .. these were just a couple of exceptions I noticed

Reply:
If these are all of them, I think it's good to go.